rca/project_styleguide/templates/patterns/atoms/modal-launcher/modal-launcher.html

```diff
@@ -0,0 +1,10 @@
+<a href="{{ item.link }}" class="modal-launcher">
+    <div class="modal-launcher__container">
+        <div class="modal-launcher__label">
+            {{ item.action }}
+        </div>
+        <div class="modal-launcher__icon-container">
+            <svg width="18" height="18" class="modal-launcher__icon"><use xlink:href="#plus"></use></svg>
```

Comment:
wonder if we should `aria-hide` to this as i don't think it adds anything for a screen reader

Reply:
Thanks, will add that into the next branch i'm working on (based off of this one)